src/app/containers/CpsAssetPageMain/index.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import CpsAssetPageMain from '.';
+import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks';
+
+const pageData = addIdsToBlocks(pidginPageData);
+
+describe('CpsAssetPageMain', () => {
+  shouldMatchSnapshot(
+    'should match snapshot',
+    <ServiceContextProvider service="news">
```

Comment:
```suggestion
    <ServiceContextProvider service="pidgin">
```

You've used pidgin data so the service should be consistent